### DISCUSSION TITLE
Prevent duplicate object to save in an array

### DISCUSSION THREAD

**Discussion by akashramtohul4** [quasarframework/quasar#10472] [CLI - PWA mode]
Hello friends ,

I have 2 inputs every time i input something i push it in an array and emits it to a parent component. the problem is each input i make create an entry in the array. i want to have only 2 entry's for each input 

can someone please help 
below is a sample code 
 https://codepen.io/akashramtohul4/pen/PojwdQN?editors=1011

Thanks

**Comment by pikil** [comment]
If I understand correctly, you do not want the array to be populated with every letter you type, right?
For that purpose, you can either use ```<q-input v-model="text1" debounce="800" />``` or set a button aside those two inputs, so a user decides by themselves when they finished typing ```<q-btn label="Change" @click="ChangeName" />```

**Reply by akashramtohul4** [reply]
yes in short it just need to update the value in the array instead of adding a new object in the array .which contains duplicate

**Reply by pikil** [reply]
@akashramtohul4 Can you use computed array|object instead?

const options = computed(() => [{ text1: text1.value, text2: text2.value }])
